### DISCUSSION TITLE
参考文献会议paper[C]后面有双斜线//如何去除

### DISCUSSION THREAD

**Discussion by Whiplash222** [tuna/thuthesis#886] [Q&A] [answered]
参考文献会议paper[C]后面有双斜线//如何去除？导师说这不符合参考文献规范

**Comment by zepinglee** [comment]
本科生还是研究生？

**Reply by Whiplash222** [reply]
本科生

**Reply by zepinglee** [reply, marked as answer]
<img width="768" alt="Screenshot 2023-05-25 at 23 19 05" src="https://github.com/tuna/thuthesis/assets/12290822/4fcba5a5-e569-4127-9743-dffc0fc710d0">

你们的格式要求是不是这份“参考文献著录格式（2019）.docx”？

按照 GB/T 7714—2015 的要求，会议论文属于“专著中析出的文献”，结果就是“[C]”后加“//”。

<img width="812" alt="Screenshot 2023-05-25 at 23 21 19" src="https://github.com/tuna/thuthesis/assets/12290822/37298c7b-0a3a-4596-9141-b790667f38c4">

**Reply by Whiplash222** [reply]
啊好的！是这个文件要求，非常感谢！！

**Reply by zepinglee** [reply]
建议你先 argue 一下。实在不行可以针对你导师的要求定制一份 bst。
